autoload/go/alternate.vim

```diff
@@ -0,0 +1,30 @@
+" By default use edit (current buffer view) to switch
+if !exists("g:go_alternate_mode")
+  let g:go_alternate_mode = "edit"
+endif
+
+" Test alternates between the implementation of code and the test code.
+function! go#alternate#Switch(bang, cmd)
+  let l:file = go#alternate#Filename(fnameescape(expand("%")))
+  if !filereadable(l:file) && !a:bang
+    echoerr "couldn't find " . file
```

Comment:
Just tested this and this should be change (we don't use it as it's not a good UX command). See:

![image](https://cloud.githubusercontent.com/assets/438920/12821558/8c024634-cb6c-11e5-9d39-f089fbceedd4.png)

This is better:

![image](https://cloud.githubusercontent.com/assets/438920/12821672/054aeece-cb6d-11e5-946c-5122543a807a.png)

You can use the following to achieve it:

```viml
    redraws! | echon "vim-go: " | echohl ErrorMsg | echon "couldn't find ".file | echohl None
```